src/main/config/config.h

```diff
@@ -53,6 +53,7 @@ typedef struct systemConfig_s {
     uint8_t configurationState;     // The state of the configuration (defaults / configured)
     uint8_t enableStickArming; // boolean that determines whether stick arming can be used
     uint8_t enableStickCommands; // boolean that determines whether stick commands can be used
+    uint8_t model_Id;
```

Comment:
Please move mode_Id to pilotConfig_t.
Also please follow the established naming convention, `modeId` in this case.

Reply:
do you want "modeId" or "modelId" as you typed?

Reply:
LOL. Dyslexia rules. `modelId`